eks-anywhere-common/Testers/Kong/test-job.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
```

Comment:
We should replace this to convert to a `CronJob` which runs with a schedule!

Reply:
this was updated to a `CronJob` please review